### DISCUSSION TITLE
How to move my domain to another repository

### DISCUSSION THREAD

**Discussion by apirak** [community/community#23283] [Other Feature Feedback, Questions, & Ideas]
<p>I success point my domain name “<a href="http://vitalsignapp.com" rel="nofollow noopener">vitalsignapp.com</a>” to “<a href="http://vitalsignapp.github.io" rel="nofollow noopener">vitalsignapp.github.io</a>” with simple HTML. Now I build a new repository with jekyll + custom Github Action to support i18n and point to ph_page branch. </p>
<p>URL: <a href="http://vitalsignapp.com" rel="nofollow noopener">vitalsignapp.com</a> -&gt; Repo: “<a href="http://vitalsignapp.github.io" rel="nofollow noopener">vitalsignapp.github.io</a>” (HTML)</p>
<p>URL: <a href="http://vitalsignapp.com/vitalsigns-website" class="inline-onebox-loading" rel="nofollow noopener">vitalsignapp.com/vitalsigns-website</a> - &gt; Repo: “vitalsigns-website”  (Jekyll+custom action)</p>
<p><strong>How to move “URL: <a href="http://vitalsignapp.com" rel="nofollow noopener">vitalsignapp.com</a>” to “Repo: vitalsigns-website” ?</strong></p>
<p>Because the “Vitalsignapp” account is the convention to <a href="http://vitalsignapp.github.io" rel="nofollow noopener">vitalsignapp.github.io</a> repository. If I remove “Repo: <a href="http://vitalsignapp.github.io" rel="nofollow noopener">vitalsignapp.github.io</a>” and add CNAME to “Repo: vitalsigs-website” the domain will point to the new repository?</p>
<p>Thank you!</p>

**Comment by darrowby385** [comment, marked as answer]
<p>Hi, <a class="mention" href="/apirak">@apirak</a>!</p>
<p>To change which repository your custom domain routes to, you need only follow these steps:</p>
<ol>
<li>
<p>Delete the existing CNAME file in your <strong>apirak/</strong>** <a href="http://vitalsignapp.github.io" rel="nofollow noopener">vitalsignapp.github.io</a>** repository.</p>
</li>
<li>
<p>Create a new CNAME file in your <strong>apirak/vitalsigns-website</strong> repository. This file should only contain a single line, your custom domain:</p>
<p>vitalsignapp.com</p>
</li>
</ol>
<p>You do not need to make any changes to your custom domain’s DNS records, nor should you. The same DNS records outlined in <a href="https://help.github.com/en/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site" rel="nofollow noopener">our documentation</a> should always be used, regardless of which specific repository your custom domain will be routed to. Repository-level routing occurs on our infrastructure, not at the DNS level.</p>
<p>Once you’ve made your changes, you might need to allow up to 10 minutes for your GitHub Pages site to rebuild. If your custom domain is reflecting your changes at that point, you might also need to clear your browser’s cache.</p>
<p>I hope that helps!</p>
<p>Friday</p>

**Reply by nabeerdar** [reply]
The custom domain `www.mydomainname.de` is already taken. If you are the owner of this domain, check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages for information about how to verify and release this domain.  
Please help, link Github provided doesn't work
Can you help me release my domain from another github acc, he transferred me in my namecheap, I also changed CNAME linking to my github, but still I'l not able to add custom domain to repo settings > code & automation > pages

**Reply by kailiang-zhong** [reply]
same as nabeerdar.

The custom domain lemoner.me is already taken. If you are the owner of this domain, check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages for information about how to verify and release this domain. 
Please help, link Github provided doesn't work. 

Can you help me release my domain from another github page, he transferred me in my namecheap, I also changed CNAME linking to my github, but still I'm not able to add custom domain to repo settings > code & automation > pages

How to move “URL:  lemoner.me” to “Repo: kailiang-zhong.github.io” ?
Please help~

**Reply by ScriptAutomate** [reply]
@nabeerdar @kailiang-zhong Not sure what solutions you ended up working with, but it sounds like you were running into domain verification issues in the process of trying to migrate. This can be an issue with GitHub if the domain is listed as configured in any repository across all of GitHub.

Verification ensures your org or user is considered the true owner, and that only your repos can then make use of the domain: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages#about-domain-verification-for-github-pages

> Verifying your domain stops other GitHub users from taking over your custom domain and using it to publish their own GitHub Pages site. Domain takeovers can happen when you delete your repository, when your billing plan is downgraded, or after any other change which unlinks the custom domain or disables GitHub Pages while the domain remains configured for GitHub Pages and is not verified.
>
> **Verifying a domain that is already taken**
>
> You may be verifying a domain you own, which is currently in use by another user or organization, to make it available for your GitHub Pages website. In this case, the domain will be immediately released from GitHub Pages websites which are owned by other users or organizations. If you are attempting to verify an already verified domain (verified by another user or organization), the release process will not be successful.
